extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -1155,4 +1156,25 @@ class HiveCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
       }
     }
   }
+
+  test("CreateDatabaseCommand/AlterDatabaseSetLocationCommand") {
+    assume(!isSparkV33OrGreater)
```

Comment:
Let's support 3.5

Reply:
Also support CreateNamespace and AlterNamespaceSetLocation command in this pr, then we don't need this.